build_functions/CMakeLists.txt

```diff
@@ -47,6 +47,203 @@ function(target_link_libraries_with_arg_prefix arg_prefix whatIsBuilding lib)
     endif()
 endfunction()
 
+########################################################################
```

Comment:
ha that's a lot of comment lines

Reply:
needed to have some delimiters about these drop-in replacements.

---
In reply to: [455885001](https://github.com/Azure/azure-c-build-tools/pull/30#discussion_r455885001) [](ancestors = 455885001)